### ISSUE TITLE
Mouse keeps moving

### ISSUE THREAD

**Issue by jastevenson303** [minetest/minetest#7186]
##### Issue type
 - Bug report
 
##### Minetest version

```
0.4.16 and 0.5.0-dev.
```

##### OS / Hardware

Operating system: Windows 10
CPU: Intel Core i5

GPU model: ?
OpenGL version: 4.5?

##### Summary

Laptop is a Windows 10 Lenovo Yoga 720-13IKB with touchscreen display.  When using an external mouse, the view will continue to move on its own, both yaw and pitch.

**Comment by paramat** [comment]
This has been reported a few times before, please search issues, i seem to remember it may be due to hardware settings.

**Comment by SmallJoker** [comment]
Duplicate of #4287 and #6245

Solution: https://github.com/minetest/minetest/issues/4287#issuecomment-240298978
The `*.manifest` file should take care of this issue, which however, is only used in MSVC-compiled binaries.

**Comment by jastevenson303** [comment]
@paramat, for the record, I did search first -- I just didn't find the issues within the approximately 3,500 tickets.

@SmallJoker, I don't understand what the *.manifest file is, or what it is you want me to do with it.  I followed the direction to the **workaround** in the comment that you linked, and the continuous movement seems to have halted; thank you.

It may be worthwhile to duplicate the workaround so others may find it easily, but I don't know.
